README.md

```diff
@@ -169,19 +169,33 @@ The Vonage Video API Reference App for React is currently supported on the lates
 
 ## Testing on Multiple Devices
 
-To test the video API across multiple devices on your local network, you can use **ngrok** to expose your frontend publicly.
+To test the video API across multiple devices on your local network, you can use **ngrok** to expose your frontend and backend publicly.
 
-1. Create an account at [ngrok](https://dashboard.ngrok.com/signup) if you haven’t already.
+1. Create an account at [ngrok](https://dashboard.ngrok.com/signup) if you haven't already.
 
 2. Follow the [Setup and Installation instructions](https://dashboard.ngrok.com/get-started/setup/) for your operating system to install and configure ngrok.
 
-3. Create a secure tunnel to your frontend using:
+3. **Start the application locally first:**
 
+    ``` bash
+    yarn dev
+    ```
+
+    Make sure both the backend server (port 3345) and frontend dev server (port 5173) are running before proceeding to the next step.
```

Comment:
How does one verify this?  Would a picture, or sample console logs, of happy path help?